src/QMCWaveFunctions/BsplineFactory/HybridAdoptorBase.h

```diff
@@ -416,7 +418,16 @@ struct HybridAdoptorBase
   // for APBC
   PointType r_image;
   // smooth function derivatives
-  RealType df_dr, d2f_dr2;
+  RealType f, df_dr, d2f_dr2;
+  /// smoothing schemes
+  enum
+  {
+    CONSISTENT = 0,
+    SMOOTHALL,
+    SMOOTHPARTIAL
+  } smooth_scheme;
+  /// smoothing function
+  int smooth_func_id;
```

Comment:
I will not approve this kind throw away of enum variable, especially as it is set far from here and used in place of what should be a proper functor or templated function. And its actually part of the "state"/recipe of the spline evaluation but its hidden here.